resources/views/components/game/claim-info.blade.php

```diff
@@ -72,7 +72,7 @@
         } else {
             // player_sessions only exist after 14 Oct 2023
             $achievement = Achievement::where('GameID', $gameId)
-                ->where('Author', $claim['User'])
+                ->where('user_id', $playerGame->user->id)
```

Comment:
This fails if the claimant has never played the game:
```
Attempt to read property "id" on null
```

Reply:
Thanks. Resolving this proved to be quite the rabbit hole.

The component currently suffers from an N+1 query problem in `master`. This is easily observable on game 20255.

I fixed in the error in latest, but I also rewrote nearly all of these queries to rely on eager-loaded model relationships in order to resolve the N+1 query issue.